room/room.go

```diff
@@ -127,6 +127,7 @@ func (t *TalkRoom) ReceiveMessages(ctx context.Context) (chan ocs.TalkRoomMessag
 			if err != nil {
 				continue
 			}
+			_ = res.Body.Close()
 			if res.StatusCode == 200 {
 				lastKnown = res.Header.Get("X-Chat-Last-Given")
 				data, err := ioutil.ReadAll(res.Body)
```

Comment:
the body is now used after being closed

Reply:
:facepalm: That was why I put in in there.